### PR TITLE
 Changed the searchModule.js file to address Issue #100

### LINKED ISSUE #100: Load more button is still present when no images are found
**Describe the bug**
When there are no images in a search query, the load more button is still rendered. Even though clicking it would not do fill the grid with empty thumbnails or show the loading animation forever, but nonetheless it should not be present in this particular case.

**To Reproduce**
1. Go to Extension
2. Search for random gibrish charachters ('eiejndinec' -> tried and tested)
3. See error

**Expected behavior**
The load more button should not be present when no images are found.

**Screenshots**
Screenshot (252)

### PR DESCRIPTION
**Pull request by gauravahlawat81**
Fixes #100 

**Description**
My initial approach was to delete the LoadMore button from the DOM tree, but that removed the Load More from further searches too. On reviewing the code again , I found that I can change the _popup.scss file and added one more element, So if there are no images to show I just add one extra class,`removeLoadMore` to the `load-more-button-wrapper` div which makes the `display:none` essentially hiding the button and removing the `removeLoadMore` when there are images to show. 

**Checklist:**
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the `master` branch of the repository.
- [x] My commit messages follow [best practices][best_practices].
- [ ] I added tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

### CODE REVIEW

**Review by makkoncept — CHANGES_REQUESTED** [review]
- You are straight away removing the node from the DOM. The problems with this approach is obvious - load-more button is not rendered when we search for anything after this and user cannot load more images! 
- Also, please fill the descriptions of the PR with the changes you have made and strategy used to solve the issue
- Also, please write better commit messages.

**Comment by gauravahlawat81** [comment]
I have made the required changes , please check and tell if it's working fine(I hope it does), will gladly work more if required.

File: src/popup/sass/components/_popup.scss [hunk 1]
```diff
@@ -141,3 +141,6 @@
   background: #cb2027;
   color: white;
 }
+.removeLoadMore{
```
Comment (makkoncept) [inline src/popup/sass/components/_popup.scss]: There is already a `display-none` rule defined in `_utilities.scss`. Please use that.

File: src/popup/searchModule.js [hunk 2]
```diff
@@ -1,13 +1,20 @@
 import { elements } from './base';
-import { unicodeToString, providerLogos, addLoadMoreButton } from './helper';
+import {
```
Comment (makkoncept) [inline src/popup/searchModule.js]: Undo these changes. Your editor must be wrapping the lines on save.

File: src/popup/searchModule.js [hunk 3]
```diff
@@ -1,13 +1,20 @@
...
+  unicodeToString, providerLogos, addLoadMoreButton,
+} from './helper';
 import { activatePopup } from './infoPopupModule';
 import { removeSpinner } from './spinner';
 // eslint-disable-next-line import/no-cycle
 import bookmarkImage from './bookmarkModule';
-import { showNotification, removeChildNodes, restoreInitialContent } from '../utils';
+import {
```
Comment (makkoncept) [inline src/popup/searchModule.js]: Undo these changes. Your editor must be wrapping the lines on save.

File: src/popup/searchModule.js [hunk 4]
```diff
@@ -1,13 +1,20 @@
...
+  showNotification, removeChildNodes, restoreInitialContent,
+} from '../utils';
 
 const Masonry = require('masonry-layout');
 
+let removed = false;
```
Comment (makkoncept) [inline src/popup/searchModule.js]: There is no need to use a variable. `node.classList.remove()` and `node.classList.add()` does not throw any exception if you are removing a class not present or if you are adding an already present class respectively.

File: src/popup/searchModule.js [hunk 5]
```diff
@@ -62,6 +69,8 @@ function showNoResultFoundMessage() {
     paragraph.textContent = 'No Images Found. Please enter a different query.';
 
     removeChildNodes(sectionContentPrimary.querySelector('.row'));
+    sectionContentPrimary.querySelector('.load-more-button-wrapper').classList.add('removeLoadMore');
```
Comment (makkoncept) [inline src/popup/searchModule.js]: All the required element selectors are present in `base.js` and are already imported in `searchModule.js`. Same is the case with `loadMoreButtonWrapper`, therefore, don't use querySelector to select it.

**Review by makkoncept — CHANGES_REQUESTED** [review]

File: src/popup/sass/components/_popup.scss (hunk 1 above)
Comment (gauravahlawat81) [reply src/popup/sass/components/_popup.scss]: This was indeed my initial approach, I was trying to use `display-none` for hiding the button, but I don't know why it wasn't working. After hours of debugging I couldn't find the bug so I had to create `removeLoadMore` which gave me the desired result.

File: src/popup/searchModule.js (hunk 2 above)
Comment (gauravahlawat81) [reply src/popup/searchModule.js]: Okay, On it

File: src/popup/searchModule.js (hunk 3 above)
Comment (gauravahlawat81) [reply src/popup/searchModule.js]: Okay

File: src/popup/searchModule.js (hunk 5 above)
Comment (gauravahlawat81) [reply src/popup/searchModule.js]: Yeah, that was redundant. I will fix that.

File: src/popup/sass/components/_popup.scss (hunk 1 above)
Comment (gauravahlawat81) [reply src/popup/sass/components/_popup.scss]: I tried again but couldn't do it. I can't really tell what's causing this behaviour , should I continue using removeLoadMore ?

File: src/popup/searchModule.js [hunk 6]
```diff
@@ -62,6 +64,10 @@ function showNoResultFoundMessage() {
     paragraph.textContent = 'No Images Found. Please enter a different query.';
 
     removeChildNodes(sectionContentPrimary.querySelector('.row'));
+    // This little code fixes the Issue #100, I am adding a new class
```
Comment (makkoncept) [inline src/popup/searchModule.js]: Can you please don't write the comment in first person. Just state what the code is doing (like the comment you wrote below).

File: src/popup/searchModule.js [hunk 7]
```diff
@@ -8,6 +8,8 @@ import { showNotification, removeChildNodes, restoreInitialContent } from '../ut
 
 const Masonry = require('masonry-layout');
 
+// let isLoadMoreRemoved = false;
```
Comment (makkoncept) [inline src/popup/searchModule.js]: Remove this line.

File: src/popup/sass/components/_popup.scss (hunk 1 above)
Comment (makkoncept) [reply src/popup/sass/components/_popup.scss]: Okay, we can improve upon that later. I have added a couple more comments, please look into it. We can merge this after that.

**Review by makkoncept — CHANGES_REQUESTED** [review]

File: src/popup/searchModule.js (hunk 7 above)
Comment (gauravahlawat81) [reply src/popup/searchModule.js]: Okay I have made the required changes
